CHANGELOG.md

```diff
@@ -99,23 +101,23 @@ Lots of tidy-ups, code-formatting and small improvements not worth mentioning :)
 ## [6.0.0-beta.3] - 2016-08-30
 
 ### Breaking changes
-- `_settings.global.scss` is renamed to `_settings.core.scss`.   
+- `_settings.global.scss` is renamed to `_settings.core.scss`.
```

Comment:
It seems you've lost some spaces here. As this is a .md file, these are important as they represent a line break. You may want to add them back in